backend/entities/session.entity.ts

```diff
@@ -0,0 +1,28 @@
+import { Entity, Column, PrimaryGeneratedColumn, ManyToOne, OneToMany } from 'typeorm';
+import { User } from './user.entity';
+import { Participant } from './participant.entity';
+import { Game } from './game.entity';
+
+@Entity()
+export class Session {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  date: Date;
+
+  @Column()
+  url: string;
+
+  @Column()
+  isActive: boolean;
+
+  @ManyToOne(() => User, user => user.sessions)
+  user: User;
```

Comment:
same as above

Reply:
@hamsof

Yes, when using TypeORM to define a many-to-one relationship between two entities (let's say A and B), you typically need to specify the relationship on both entities.

In entity A, you'll define the many-to-one relationship using the @ManyToOne decorator, specifying the target entity (B in this case). This decorator should go on the property of entity A that represents the relationship with entity B.

In entity B, you'll define the inverse side of the relationship using the @OneToMany decorator. This decorator should also go on a property of entity B, representing the collection of entities A related to B.
Credits: ChatGPT